### DISCUSSION TITLE
My last day with Etsy...

### DISCUSSION THREAD

**Discussion by etsyachristensen** [etsy/open-api#768] [Show and tell]
I am taking in my last day here preparing as my contract comes to close with Etsy. It's a sad day for me as I have enjoyed every moment of the last 1 year, 7 months, and 18 days working on this great project and helping the company _Keep Commerce Human_.

It's been a privilege to help all of you through some of the trials and tribulations that come with launching and migrating to a new version of an API. While I know we haven't gotten everything right from the start, we've tried very hard to provide what we could with the restrictions we've had (time, technical, practical and legal). I appreciate the immense patience you've shown. You will likely start to see 4 new people over the coming weeks jumping into the system to ask questions of you and help refine the issue posts down to more efficiently accomplish the task. I'm confident that my team members stepping into the OpenAPI management role will bring a renewed push for development excellence.

As I search for what's next in my career path, I do hope to one day return to Etsy and maybe I'll have the chance again to serve you all. I'll still be lurking in my personal account. (Giving some thought to trying my hand at building a PHP package for the OpenAPI).

Anyway, it has been an honor to work with you all. Thank you again!

- Andrew Christensen

**Comment by assaflei** [comment]
Best of luck in your next position @etsyachristensen

**Reply by sowsinsk** [reply]
Best of luck, @etsyachristensen

**Reply by Mad4F1** [reply]
Best of luck, @etsyachristensen, and thanks for your great work. It's much appreciated!

**Comment by kbasali** [comment]
@etsyachristensen Thank you so much for your patience and help.

**Comment by Gareth064** [comment]
Sorry to see you're leaving @etsyachristensen . You have been a real constant in this journey so far. Good luck with your next adventure.

**Comment by baahar** [comment]
Thanks for all the quick responses and best of luck !

**Comment by Makebuildmodify** [comment]
@etsyachristensen  Thanks!
